app/models/concerns/description_sanitation.rb

```diff
@@ -0,0 +1,11 @@
+module DescriptionSanitation
+  extend ActiveSupport::Concern
+
+  included do
+    before_save :sanitize_description
+  end
+
+  def sanitize_description
+    description['en'] = Sanitize.fragment(description['en'], Sanitize::Config::RESTRICTED)
```

Comment:
This should be set up to sanitize all the fields like so:

```suggestion
    description.transform_values { |desc| Sanitize.fragment(desc, Sanitize::Config::RESTRICTED) }
```